.github/get-docker-tag.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Calculate hash from the following files. This hash is used to tag the docker images.
+# Any change in these files will result in a new docker image build
+DOCKERFILE_HASH_FILES=".github/Dockerfile.base .github/Dockerfile.ci .github/Dockerfile.ci env/CMakeLists.txt"
+DOCKERFILE_HASH=$(md5sum $DOCKERFILE_HASH_FILES | md5sum | cut -d ' ' -f 1)
```

Comment:
Ok, this will make slightly longer tags, but it's ok we don't hit any length limitations
dt-e931311c5edde3f3df8df7b8bf42e9dd
dt-8d11f16163457c4007e6c21d000dee14c80887131e4dce6a3b34f106e25ea344